$MirOS: ports/devel/libtool/patches/patch-Makefile_am,v 1.1 2005/12/16 14:52:21 tg Exp $
--- Makefile.am.orig	Thu Jan  1 00:00:00 1970
+++ Makefile.am	Mon Jun 12 19:35:20 2006
@@ -24,7 +24,7 @@ CLEANFILES = libtool libtoolize ltmain.s
 # These are required by libtoolize and must be executable when installed.
 # Since _SCRIPTS gets the program transform applied we make them
 # executable by hand
-pkgdata_DATA = config.guess config.sub install-sh ltmain.sh
+pkgdata_DATA = config.guess config.sub install-sh ltmain.sh m4salt.inc m4sugar.inc
 
 # This macro file should be visible to Automake's aclocal.
 aclocal_DATA = $(aclocal_macros)
@@ -63,7 +63,7 @@ libtoolize: $(srcdir)/libtoolize.in $(to
 	CONFIG_FILES=$@ CONFIG_HEADERS= $(SHELL) $(top_builddir)/config.status
 	chmod +x $@
 
-$(srcdir)/ltmain.sh: $(srcdir)/ltmain.in $(TSDEPS)
+no-$(srcdir)/ltmain.sh: $(srcdir)/ltmain.in $(TSDEPS)
 	-rm -f ltmain.shT
 	date=`$(SHELL) $(srcdir)/mkstamp < $(srcdir)/ChangeLog` && \
 	sed -e 's/@''PACKAGE@/@PACKAGE@/' -e 's/@''VERSION@/@VERSION@/' \
